topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1706,108 +1756,62 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >
 {: .hands_on}
 
-Galaxy is now configured with an admin user, a database, and a place to store data. Additionally we've immediately configured the mules for production Galaxy serving. So we're ready to set up systemd which will manage the Galaxy processes! Launching Galaxy by hand is not a good use of your time, so we will immediately switch to a process manager for that, [systemd](https://freedesktop.org/wiki/Software/systemd/).
+Galaxy is now configured with an admin user, a database, and a place to store data. We've immediately configured standalone Galaxy servers to handle jobs and workflow scheduling. Additionally, Galaxy's process manager, Gravity, has written out [systemd](https://freedesktop.org/wiki/Software/systemd/) configuration files called *service units* that are used to define how to start and stop Galaxy's various services. Your Galaxy server should now be running!
 
-> <hands-on-title>systemd</hands-on-title>
->
-> 1. Configure systemd in `group_vars/galaxyservers.yml` file:
->
->    {% raw %}
->    ```diff
->    --- a/group_vars/galaxyservers.yml
->    +++ b/group_vars/galaxyservers.yml
->    @@ -57,3 +57,6 @@ galaxy_config:
->             pools:
->               - job-handler
->               - workflow-scheduler
->    +
->    +# systemd
->    +galaxy_manage_systemd: true
->    {% endraw %}
->    ```
->    {: data-commit="Setup systemd variables"}
->
->    > <tip-title>Using Galaxy 21.01?</tip-title>
->    > For versions of Galaxy older than 22.05, using `gravity` was not the default. There you'll need to set these additional variables:
->    > {% raw %}
->    > ```diff
->    > +galaxy_manage_gravity: "{{ false if __galaxy_major_version is version('22.01', '<') else true }}"
->    > +galaxy_systemd_mode: "{{ 'mule' if __galaxy_major_version is version('22.01', '<') else 'gravity' }}"
->    > ```
->    > {% endraw %}
->    {: .tip}
->
-> 2. Run the playbook
+> <hands-on-title>Status Check</hands-on-title>
 >
->    > <code-in-title>Bash</code-in-title>
->    > ```bash
->    > ansible-playbook galaxy.yml
->    > ```
->    > {: data-cmd="true"}
->    {: .code-in}
->
-> 3. Log in and check the status with `sudo systemctl status galaxy`
+> 1. Log in and check the status with `sudo galaxyctl status`
 >
 >    > <code-in-title>Bash</code-in-title>
 >    > ```bash
->    > sudo systemctl status galaxy
+>    > sudo galaxyctl status
```

Comment:
I went back and forth over this, the Gravity syntax is a bit shorter, the equivalent systemd syntax would be `systemctl status 'galaxy*'`, likewise instead of `galaxyctl status gunicorn` you have `systemctl status galaxy-gunicorn`. But I'm totally fine with pushing everyone to just interact with systemd/journald natively.

In the changes to the role (unpublished) I drop a wrapper around `galaxyctl` into `/usr/local/bin` that sets `$GRAVITY_CONFIG_FILE` appropriately so that you can call it from anywhere. Seemed like a useful convenience.

Reply:
Both? Both is good. Show galaxyctl first, and then show the systemd units that get created? I think it's useful to show those as well for e.g. things like memory limits? And just to show the connection a bit? (We could just look at one, maybe, instead of all)